diff --git a/node_modules/node-record-lpcm16/recorders/sox.js b/node_modules/node-record-lpcm16/recorders/sox.js
index 8ebfe2b..1c8b255 100644
--- a/node_modules/node-record-lpcm16/recorders/sox.js
+++ b/node_modules/node-record-lpcm16/recorders/sox.js
@@ -2,7 +2,8 @@ module.exports = (options) => {
   const cmd = 'sox'
 
   let args = [
-    '--default-device',
+    //'--default-device',
+    '--type', 'waveaudio', '0', // audio type
     '--no-show-progress', // show no progress
     '--rate', options.sampleRate, // sample rate
     '--channels', options.channels, // channels
